src/SubGraphFunctions.jl

```diff
@@ -53,10 +53,12 @@ Related
 
 getVariableIds
 """
-function buildSubgraphFromLabels(dfg::G, syms::Vector{Symbol}) where G <: AbstractDFG
+function buildSubgraphFromLabels(dfg::G, 
+                                 syms::Vector{Symbol}, 
+                                 destType::Type{<:AbstractDFG}=LightGraphsDFG ) where G <: AbstractDFG
 
   # data structure for cliq sub graph
-  cliqSubFg = initfg()
+  cliqSubFg = initfg(destType)
```

Comment:
I made a small change here, hope that is okay?

Reply:
work on copy for now and yes keeping same type is probably better, you're right.